ios/RNSScreen.h

```diff
@@ -58,6 +58,13 @@ NS_ASSUME_NONNULL_BEGIN
 @property (nonatomic) RNSScreenStackPresentation stackPresentation;
 @property (nonatomic) RNSScreenSwipeDirection swipeDirection;
 @property (nonatomic) RNSScreenReplaceAnimation replaceAnimation;
+
+// Props controlling UISheetPresentationController
+@property (nonatomic) RNSScreenDetentType sheetAllowedDetents;
+@property (nonatomic) RNSScreenDetentType sheetLargestUndimmedDetent;
+@property (nonatomic) BOOL isSheetGrabberVisible;
```

Comment:
`is` is not necessary, see other BOOL props

Reply:
[fe7b019](https://github.com/software-mansion/react-native-screens/pull/1649/commits/fe7b0199a286788cf5d9dc51f42595993820eb4a)